recipes/divvier/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -eu -o pipefail
+
+echo ${PREFIX}
+mkdir -p ${PREFIX}/bin
+make
```

Comment:
You can use `make CC=$CC CPP=$CXX` and get rid of the patch

Reply:
Thanks for the help. Didn't know about this.